project.clj

```diff
@@ -139,7 +139,9 @@
                                        :main-namespace "puppetlabs.puppetdb.core"
                                        :create-varlib true}
                                 :config-dir "ext/config/pe"}
-                  :version ~pe-pdb-version
+                  :plugins [[puppetlabs/lein-ezbake "0.2.8"
+                              :exclusions [org.clojure/clojure]]]
```

Comment:
To be clear, my point about VPN is specifically about trying to retrieve the jar, not so much running the build after that fact. That is, non-staff would hit problems doing anything (even non build stuff) with our project if its at the top level, and they'd have to comment out the dep to go any further.